frontends/web/src/components/view/view.module.css

```diff
@@ -77,6 +77,10 @@
         padding-bottom: var(--space-half);
         padding-top: 2vh;
     }
+
+    .verticallyCentered .inner {
+        flex-grow: 0;
+    }
```

Comment:
Had to add this in order to mitigate this bug (I think we missed it):

**Before,** when using `View` with `verticallyCentered` on mobile, it doesn't vertically center:

![Screenshot 2023-02-21 at 10 15 32](https://user-images.githubusercontent.com/28676406/220301630-c12c205e-9243-435a-8efb-a6a0a7e09e04.png)

**After**
![Screenshot 2023-02-21 at 10 15 43](https://user-images.githubusercontent.com/28676406/220301743-90b070f2-0826-4457-be9a-3417f08541d7.png)

Reply:
yes View still is not perfect and needs more attention. I looked into that as well yesterday and made a similar change https://github.com/digitalbitbox/bitbox-wallet-app/commit/b1079769e23bf01df574bd8165941b6033fa7802